### PR TITLE
Changed filters UI to be dropdown menu

### PR DESCRIPTION
**Pull request by graceehuu**
changed suggested filters UI to be a dropdown menu, centered horizontally
now user can only select at most 1 filter

### CODE REVIEW

File: capstone/src/main/webapp/script.js [hunk 1]
```diff
@@ -16,13 +16,13 @@ window.addEventListener('DOMContentLoaded', (event) => {
   if (document.getElementById('map')) {
     createMap();
   }
-  if (document.getElementById('filters-form')) {
-    getInputFilters();
-  }
+//   if (document.getElementById('filters-form')) {
+//     getInputFilters();
```
Comment (gannerr) [inline capstone/src/main/webapp/script.js:20]: are we saving this for later use?

Comment (graceehuu) [reply capstone/src/main/webapp/script.js:20]: good catch!

File: capstone/src/main/webapp/script.js [hunk 2]
```diff
@@ -327,24 +317,30 @@ function isStringEmpty(str) {
 }
 
 /** post request params to send a POST request using fetch() */
-const requestParamPOST = {
-  method: 'POST',
-  headers: {
-    'Content-Type': 'application/json',
-  },
-};
+// const requestParamPOST = {
```
Comment (casanted) [inline capstone/src/main/webapp/script.js:320]: Sorry if I missed this before, but why are we commenting this part out?

File: capstone/src/main/webapp/script.js [hunk 3]
```diff
@@ -327,24 +317,30 @@ function isStringEmpty(str) {
...
+  return reviewsEntities;
 }
 
 /** send POST request to Cloud Natural Language API for entity recognition */
-function getEntities(message) {
-  const url = '/nlp-entity-recognition?message=' + message;
-  fetch(url, requestParamPOST).then((response) => response.json()).then((entities) => {
-    const businessTags = entities;
+function getEntities(messages) {
+  const url = '/nlp-entity-recognition?messages=' + messages;
```
Comment (casanted) [inline capstone/src/main/webapp/script.js:335]: Oh I see you put it back in here. But then why don't we just delete the previous instance?

Comment (graceehuu) [reply capstone/src/main/webapp/script.js:335]: oop forget to delete good catch!

File: capstone/src/main/webapp/script.js (hunk 2 above)
Comment (terencedignon) [reply capstone/src/main/webapp/script.js:320]: +1

**Comment by graceehuu** [comment]
remaining linter fixes are features that have not yet been implemented or address parts that are removed from moving some api calls from client side to server side

**Pull request merged**